src/devices/MemoryLCD/MemoryLcd.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
I'm curious, any reason why you went with 3.1 as opposed to 2.1? I ask because iot.device.bindings ships for 2.1 and 5.0, so that means that this binding would only be available for the 5.0 version so ideally we would want to be in both if possible.